test/e2e/custom_tool_test.go

```diff
@@ -24,9 +24,8 @@ func TestCustomToolWithGitCreds(t *testing.T) {
 				},
 			},
 		).
-		CustomCACertAdded().
 		// add the private repo
-		HTTPSRepoURLAdded().
+		HTTPSInsecureRepoURLAdded().
```

Comment:
This test verifies cert management functionality. I would like to investigate why it is failing instead of switching to insecure

Reply:
this test does not need to do that, as it's only meant to test custom tool env

Reply:
Please see #2097 for the fix - I think this PR can be closed unmerged.